west.yml

```diff
@@ -124,7 +124,7 @@ manifest:
       revision: 6489e771e9c405f1763b52d64a3f17a1ec488ace
       path: modules/lib/cmsis-dsp
     - name: cmsis-nn
-      revision: 0c8669d81381ccf3b1a01d699f3b68b50134a99f
+      revision: ea987c1ca661be723de83bd159aed815d6cbd430
```

Comment:
Based on what we do for other modules (e.g. TF-M), it would make more sense to merge the CMSIS-NN v6.0.0 tag into the existing `zephyr` branch than to create and reference a separate `zephyr-v6.0.0` branch.

Note that merge commits are allowed in the CMSIS-NN module repo.

Reply:
What makes more sense is up to the repo maintainer to decide. For the other modules I control I create branches per upstream release, as I see more value in having a clean diff (that is possible to review) from the upsteam release than a linear commit history (e.g. https://github.com/zephyrproject-rtos/loramac-node).

Reply:
> What makes more sense is up to the repo maintainer to decide

Sure, as long as the maintainer sticks with only one method.

> For the other modules I control I create branches per upstream release, as I see more value in having a clean diff (that is possible to review) from the upsteam release than a linear commit history

That makes sense; but, that does not seem to be how upstream syncs are meant to be handled for the particular module (especially, looking at the branch name) -- for example, what is the `zephyr` branch supposed to be after `zephyr-v6.0.0` branch is created?

If you, as the maintainer of the CMSIS-NN module, decide to do this, it would only make sense to rename the existing `zephyr` branch accordingly.

On a side note, maybe we need a place to properly document how every module handles upstream syncs so that we do not end up with a mess like `zephyr`, `zephyr-v2.6` and `v4.7.0-zephyr`, as seen in the `loramac-node` module repository.